extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ChangeEvent } from 'react';
+import { TextInput } from '@primer/react';
+import { SearchIcon } from '@primer/octicons-react';
+
+interface RepositoriesSearchProps {
+  filterValue: string;
+  setFilterValue: (value: string) => void;
+}
+
+const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchProps) => {
+  return <>
+    <TextInput
+      block
+      sx={{
+        backgroundColor: 'var(--vscode-editor-background);',
+        color: 'var(--vscode-editor-foreground);'
+      }}
+      className="vscode-codeql__repositories-search"
```

Comment:
This class is added to add some space to the top. I've generally used `<VerticalSpace />` for that e.g. `<VerticalSpace size={2} />`

Reply:
Sure, I can change that. Just wondering what the benefit of using extra dom for adding space instead of using css.

Reply:
IMO it's much easier to follow/reason with rather than poking with margins and padding (and dealing with conflicting margins, different context/layout etc.)